posts/2025-01-09-Rust-1.84.0.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+title: "Announcing Rust 1.84.0"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.84.0. Rust is a programming language empowering everyone to build reliable and efficient software.
+
+If you have a previous version of Rust installed via `rustup`, you can get 1.84.0 with:
+
+```console
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`](https://www.rust-lang.org/install.html) from the appropriate page on our website, and check out the [detailed release notes for 1.84.0](https://doc.rust-lang.org/stable/releases.html#version-1840-2025-01-09).
+
+If you'd like to help us out by testing future releases, you might consider updating locally to use the beta channel (`rustup default beta`) or the nightly channel (`rustup default nightly`). Please [report](https://github.com/rust-lang/rust/issues/new/choose) any bugs you might come across!
+
+## What's in 1.84.0 stable
+
+### Cargo can use toolchain version for library version selection
+
+1.84.0 stabilizes the minimum supported Rust version (MSRV) aware resolver,
```

Comment:
```suggestion
1.84.0 stabilizes the MSRV-aware resolver,
```

The acronym is explained by the link later in the sentence, and avoiding the expansion *here* allows hyphenating "MSRV-aware".

Reply:
Typically you want to introduce an acronym *before* you use it, right? I think if we change this we'd lose any direct equivalence statement (i.e., "expanded" = "MSRV") -- you'd have to click through to figure it out. MSRV doesn't feel *that* well known a term to me, so I think writing it out is better for the first reference.